scripts/tasks.sh

```diff
@@ -469,4 +474,98 @@ EOF
     done < <(grep ':6789:/' /proc/mounts | grep -v globalmount | awk '{ print $2 }' | awk -F '/' '{ print $6 }')
 }
 
+function migrate_pvcs() {
+    export KUBECONFIG=/etc/kubernetes/admin.conf
+
+    # get params - specifically need airgap as that impacts binary downloads and skip-rook-health-checks to skip rook health checks
+    shift # the first param is migrate_pvcs/migrate-pvcs
+    while [ "$1" != "" ]; do
+        _param="$(echo "$1" | cut -d= -f1)"
+        _value="$(echo "$1" | grep '=' | cut -d= -f2-)"
+        case $_param in
+            airgap)
+                AIRGAP="1"
+                ;;
+            skip-rook-health-checks)
+                SKIP_ROOK_HEALTH_CHECKS="1"
+                ;;
+            skip-longhorn-health-checks)
+                SKIP_LONGHORN_HEALTH_CHECKS="1"
+                ;;
+            *)
+                echo >&2 "Error: unknown parameter \"$_param\""
+                exit 1
+                ;;
+        esac
+        shift
+    done
+
+    download_util_binaries
+
+    # check that rook-ceph is healthy
+    CEPH_HEALTH_DETAIL=$(kubectl exec -n rook-ceph deployment/rook-ceph-operator -- ceph health detail)
```

Comment:
There are some customers on Rook 1.4.3+ that may be interested in migrating to Longhorn. For those versions you'd have to exec into `rook-ceph-tools`.

Reply:
I'll make that exec destination conditional